scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/MultiPartySparkQueryExample.scala

```diff
@@ -25,37 +25,33 @@ object MultiPartySparkQueryExample extends Supportive {
   def main(args: Array[String]): Unit = {
 
     val ppmlArgs = Map(
-      "spark.bigdl.enableMultiKms" -> "true",
-      "spark.bigdl.kms.amyKms.type" -> "EHSMKeyManagementService",
-      "spark.bigdl.kms.amyKms.ip" -> args(0),
-      "spark.bigdl.kms.amyKms.port" -> args(1),
-      "spark.bigdl.kms.amyKms.appId" -> args(2),
-      "spark.bigdl.kms.amyKms.apiKey" -> args(3),
-      "spark.bigdl.kms.bobKms.type" -> "EHSMKeyManagementService",
-      "spark.bigdl.kms.bobKms.ip" -> args(4),
-      "spark.bigdl.kms.bobKms.port" -> args(5),
-      "spark.bigdl.kms.bobKms.appId" -> args(6),
-      "spark.bigdl.kms.bobKms.apiKey" -> args(7)
+      "spark.bigdl.primaryKey.AmyPK.plainText" -> args(0),
```

Comment:
Too many args. Use args parser maybe more user friendly.

Reply:
Parser is not adaptive to multi-party scenario, e.g. 2 input/key paths are needed while only one is defined in parser, while geting from args directly in multi-party example is more scalable, which can be a reference for user to write his application.